modules/ROOT/pages/ui-localization.adoc

```diff
@@ -8,7 +8,4 @@ include::partial$configuration/language.adoc[]
 
 include::partial$configuration/language_url.adoc[]
 
-////
-undocumented
-include::partial$configuration/language_load.adoc[]
-////
+include::partial$configuration/language_load.adoc[]
```

Comment:
This seems to be live in 6, why is it showing up as added in 7 🤔

Reply:
yeah not sure, it was merged directly to `release/6` on the 20th dec https://github.com/tinymce/tinymce-docs/pull/3027/files

Reply:
This branch was made from release/docs-6, but the staging/docs-7 branch was made from staging/docs-6 so it is missing the live changes.